uwp/Pdf-Viewer/Concepts-and-Features/Working-with-free-text-callout-annotation.md

```diff
@@ -0,0 +1,212 @@
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationSettings.AutoSizeFreeTextInputBox= true;
+
+{% endhighlight %}
+
+### Setting the default callout line segment count
+
+You can set the default callout line segment count of the free text callout annotations by using the [`CalloutLineSegmentCount`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.PdfViewerFreeTextCalloutAnnotationSettings.html#Syncfusion_Windows_PdfViewer_PdfViewerFreeTextCalloutAnnotationSettings_CalloutLineSegmentCount) property. By default, the free text callout annotation is added with two line segments.
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationSettings.CalloutLineSegmentCount = CalloutLineSegmentCount.One;
+
+{% endhighlight %}
+
+### Changing the properties of a selected free text callout
+
+You can change the properties of a selected free text callout annotation or remove it by right clicking it and choosing the desired property from the displayed options.
+ 
+ ![customtoolbarimage](images/image7.png)
+
+## Detecting the movement of free text callout annotations
+
+The movement made in free text callout annotations can be detected using the `FreeTextCalloutAnnotationMoved` event. The property values before and after the change can be obtained using the [`FreeTextCalloutAnnotationMovedEventArgs`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.FreeTextCalloutAnnotationMovedEventArgs.html) parameter of the event’s handler.
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationMoved += PdfViewer_FreeTextCalloutAnnotationMoved;
+
+private void PdfViewer_FreeTextCalloutAnnotationMoved (object sender, FreeTextCalloutAnnotationMovedEventArgs e)
+{
+         //Obtain the new bounds of entire annotation after change
+	Rect newBounds =  e.NewBounds;
+	// Obtain the old bounds of entire annotation after change
+	Rect oldBounds = e.OldBounds;
+	// Obtain the new bounds of callout text after change
+	Rect newCalloutBounds = e.NewCalloutBounds;
+ 	// Obtain the old bounds of callout text after change
+	Rect oldCalloutBounds = e.OldCalloutBounds;
+}
+
+{% endhighlight %}
+
+## Detecting the resize of free text callout annotations
+
+The resize of free text callout annotations can be detected using the `FreeTextCalloutAnnotationResized` event. The property values before and after the change can be obtained using the [`FreeTextCalloutAnnotationResizedEventArgs`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.FreeTextCalloutAnnotationResizedEventArgs.html) parameter of the event’s handler.
```

Comment:
The resizing of the free text callout annotations can be detected using the `FreeTextCalloutAnnotationResized` event. The property values before and after the change can be obtained using the [`FreeTextCalloutAnnotationResizedEventArgs`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.FreeTextCalloutAnnotationResizedEventArgs.html) parameter of the event’s handler.